.github/workflows/build.yml

```diff
@@ -212,13 +222,74 @@ jobs:
           npm run test:generate
           npm run test --shard=${{ matrix.shard }}
 
+  console:
```

Comment:
I think this flow could be optimized to use the tgz files directly:

1. Clone Console, setup node and pnmp
1. Download the wing and wingsdk tgz artifacts
1. pnpm link both tgz packages